react/components/organisms/chat/chat.js

```diff
@@ -89,6 +85,7 @@ export class Chat extends PureComponent {
         };
 
         this.props.onNewMessage(message);
+        this.scrollToEnd();

```

Comment:
It's a detail, but we probably should do this before `onNewMessage`. The reasoning is: do everything you have to do before calling the `I'm done` callback.

Reply:
My reasoning was that if for some reason `onNewMessage` fails, `scrollToEnd` would not be called. Changing